pkg/kube/client.go

```diff
@@ -324,6 +339,14 @@ func newClientInternal(clientFactory util.Factory, revision string) (*client, er
 	}
 	c.dynamicInformer = dynamicinformer.NewDynamicSharedInformerFactory(c.dynamic, resyncInterval)
 
+	c.xnsInformerFactory = xnsinformers.NewSharedInformerFactoryWithOptions(
+		c.dynamic,
+		resyncInterval,
+		xnsinformers.WithNamespaces([]string{metav1.NamespaceAll}), // TODO: Pass namespaces.
```

Comment:
Is this TODO going to be fixed in this PR?

Reply:
No, but I guess I was considering this part of the MAISTRA-2051 ticket. We can create a different one if you want though. Assuming this is about JIRA.

Reply:
No strong opinion here, I'm just worried we could forget about it, so, better document it at some Jira